cmd/incus/storage_volume.go

```diff
@@ -951,12 +951,12 @@ func (c *cmdStorageVolumeEdit) Command() *cobra.Command {
 	cmd.Use = usage("edit", i18n.G("[<remote>:]<pool> [<type>/]<volume>"))
 	cmd.Short = i18n.G("Edit storage volume configurations as YAML")
 	cmd.Long = cli.FormatSection(i18n.G("Description"), i18n.G(
-		`Edit storage volume configurations as YAML`))
-	cmd.Example = cli.FormatSection("", i18n.G(
-		`Provide the type of the storage volume if it is not custom.
-Supported types are custom, image, container and virtual-machine.
+		`Edit storage volume configurations as YAML
 
-incus storage volume edit [<remote>:]<pool> [<type>/]<volume> < volume.yaml
+Provide the type of the storage volume if it is not custom.
+Supported types are custom, image, container and virtual-machine.`))
+	cmd.Example = cli.FormatSection("", i18n.G(
+		`incus storage volume edit [<remote>:]<pool> [<type>/]<volume> < volume.yaml
```

Comment:
Examples should be functional, so not include variables or optional entries.

So in this case we'd probably have to change it to something like:
```
incus storage volume edit default foo < volume.yaml
  Update the foo custom storage volume in the default pool with YAML data from volume.yaml
```